pkg/kubelet/cm/cpumanager/state/checkpoint.go

```diff
@@ -53,7 +53,7 @@ type CPUManagerCheckpointV2 = CPUManagerCheckpoint
 
 // NewCPUManagerCheckpoint returns an instance of Checkpoint
 func NewCPUManagerCheckpoint() *CPUManagerCheckpoint {
-	//lint:ignore unexported-type-in-api user-facing error message
+	//nolint:staticcheck // unexported-type-in-api user-facing error message
```

Comment:
We should open a help-wanted to revisit these all and see if there are "real" fixes or if the nolint is permanent (I see both in here)